packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -147,10 +146,10 @@ import { Chart, ChartArea, ChartAxis, ChartGroup, ChartThemeColor } from '@patte
       </ChartGroup>
     </Chart>
   </div>
-</div>
+)
 ```
 
-```js title=Multi-color-(unordered)-bottom-left-aligned-legend-and-responsive-container
+```js title=Multi--color-(unordered)-bottom--left--aligned-legend-and-responsive-container
```

Comment:
The title was "Multi-color (unordered) chart with bottom-left aligned legend and responsive container"

Also, "bottom-left-aligned" does not need a dash, it's just "bottom-left"